storybook/storybook-css/src/components/Link/Link.stories.tsx

```diff
@@ -0,0 +1,197 @@
+      type: { name: 'string', required: false },
+      table: {
+        category: 'HTML attribute',
+        defaultValue: { summary: '' },
+      },
+    },
+    target: {
+      name: 'target',
+      type: { name: 'string', required: false },
+      table: {
+        category: 'HTML attribute',
+        defaultValue: { summary: '' },
+      },
+    },
+  },
+  args: {
+    children: 'Linktekst',
+    href: '#',
+  },
+} satisfies Meta<typeof Link>
+
+export default meta
+
+export const Standalone: Story = {
+  name: 'Standalone',
+  parameters: {
+    docs: {
+      source: {
+        language: 'html',
+        code: '<a href="#" class="amsterdam-link">Linktekst</a>',
+      },
+    },
+  },
+}
+
+export const StandaloneBold: Story = {
+  name: 'Standalone Bold',
+  parameters: {
+    docs: {
+      source: {
+        language: 'html',
+        code: '<a href="#" class="amsterdam-link amsterdam-link--standalone-bold">Linktekst</a>',
+      },
+    },
+  },
+  args: {
+    variant: 'standaloneBold',
+  },
+}
+
+export const Inline: Story = {
+  name: 'Inline',
+  parameters: {
+    docs: {
+      source: {
+        language: 'html',
+        code: '<p class="amsterdam-paragraph">Jouw <a href="#" class="amsterdam-link amsterdam-link--inline">typograaf</a> biedt mij zulke exquise schreven!</p>',
+      },
+    },
+  },
+  args: {
+    children: 'typograaf',
+    href: '#',
+    variant: 'inline',
+  },
+  decorators: [
+    (Story) => (
+      <Paragraph>
+        Jouw <Story /> biedt mij zulke exquise schreven!
+      </Paragraph>
+    ),
+  ],
+}
+
+export const InList: Story = {
+  name: 'InList',
+  parameters: {
+    docs: {
+      source: {
+        language: 'html',
+        code: `
+<a href="#" class="amsterdam-link amsterdam-link--in-list">
+  <span class="amsterdam-icon amsterdam-icon-size-6">
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" aria-hidden="true" focusable="false">
+      <path fill-rule="evenodd" d="m9.757 32-2.9-2.91L19.937 16 6.857 2.91 9.757 0l16 16z"></path>
+    </svg>
+  </span>
+  Linktekst
+</a>
+        `,
+      },
+    },
+  },
+  args: {
+    children: 'Linktekst',
+    href: '#',
+    variant: 'inList',
+  },
+}
+
+/**
+ * Link to use on a dark background
+ */
+export const Dark: Story = {
+  name: 'Dark',
+  parameters: {
+    docs: {
+      source: {
+        language: 'html',
+        code: '<a href="#" class="amsterdam-link amsterdam-link--backgorund-color-dark">Linktekst</a>',
+      },
+    },
+  },
+  args: {
+    onBackground: 'dark',
+  },
+  decorators: [
+    (Story) => (
+      <div style={{ background: '#004699', display: 'inline', padding: '16px' }}>
+        <Story />
+      </div>
+    ),
+  ],
+}
+
+/**
+ * Link to use on a light background
+ */
+export const Light: Story = {
+  name: 'Light',
+  parameters: {
+    docs: {
+      source: {
+        language: 'html',
+        code: '<a href="#" class="amsterdam-link amsterdam-link--color-white">Linktekst</a>',
```

Comment:
```suggestion
        code: '<a href="#" class="amsterdam-link amsterdam-link--color-light">Linktekst</a>',
```

Reply:
It's actually `amsterdam-link amsterdam-link--on-background-light` forgot to change it.